guid_renderer.py

```diff
@@ -0,0 +1,30 @@
+from binaryninja import DataRenderer, DisassemblyTextLine, InstructionTextToken, InstructionTextTokenType, BinaryReader, \
+    log_alert, BinaryView, VoidType, HighLevelILCallSsa
+import binaryninja
+
+
+class EfiGuidDataRenderer(DataRenderer):
```

Comment:
Recommend running pylint against this and adding doc strings to classes and exported functions. I know a lot of the existing code isn't linted or PEP8 compliant, but I've been doing my best to use good python practices going forward. See `moduletype.py` for example.